### PR TITLE
Version up current workfile in ayon menu

### PR DESCRIPTION
**Pull request by moonyuet**
## Changelog Description
This PR is to add version 
Resolve https://github.com/ynput/ayon-blender/issues/75

## Additional review information
Should we introduce the validator or integrator to check/save workfile with version?

## Testing notes:
1. Following the testing step in 75 to enable version up workfile
2. Launch Blender and `version up current workfile` would show up in the menu

### CODE REVIEW

**Review by LiborBatek — CHANGES_REQUESTED** [review]
It works in general (and perform incremental save) ...but there are few things to consider

First the menu item does have wrong tool tip as seen on the pic below:


Screenshot 2024-12-11 101227

second I would follow the maya one by its naming and also put it as a first item in the AYON menu? I quite like that fact

Screenshot 2024-12-11 101134

Lastly (optional) ```Blender``` natively offers ```Incremental Save``` via ```Ctrl+ALT+S``` and it would be cool to use it for AYON up version action too...so it would be as natural as anytime when in blender even without our integration... not sure tho if possible

**Review by LiborBatek — CHANGES_REQUESTED** [review]
Its better on top and with hotkey...which is cool but

image

Add correct name ```Version Up Workfile``` and Correct tooltip as now its off ```Launch Avalon Publisher```... to something like ```perform incremental save``` or so

File: client/ayon_blender/api/ops.py [hunk 1]
```diff
@@ -423,6 +435,24 @@ def draw(self, context):
...
         )
         context_label_item.enabled = False
+        project_name = get_current_project_name()
+        project_settings = get_project_settings(project_name)
+        if project_settings["core"]["tools"]["ayon_menu"].get(
+            "version_up_current_workfile"):
+                layout.separator()
+                layout.operator(
+                    VersionUpWorkfile.bl_idname,
+                    text="Version Up Current Workfile"
```
Comment (moonyuet) [inline client/ayon_blender/api/ops.py:445]:
```suggestion
                    text="Version Up Workfile"
```

**Review by LiborBatek — APPROVED** [review]
Works like charm now!

image

**Pull request merged**